packages/adapter-sveltekit/src/hooks/sliceSimulator-setup-read.ts

```diff
@@ -0,0 +1,203 @@
+import type {
+	SliceMachineContext,
+	SliceSimulatorSetupReadHook,
+	SliceSimulatorSetupStep,
+} from "@slicemachine/plugin-kit";
+import { source } from "common-tags";
+import { createRequire } from "node:module";
+import fetch, { Response } from "node-fetch";
+
+import type { PluginOptions } from "../types";
+
+const REQUIRED_DEPENDENCIES = ["@prismicio/svelte"];
+
+type Args = SliceMachineContext<PluginOptions>;
+
+const createStep1 = async ({
+	project,
+	helpers,
+}: Args): Promise<SliceSimulatorSetupStep> => {
+	const require = createRequire(project.root);
+
+	return {
+		title: "Install packages",
+		description: "The simulator requires some dependencies.",
+		// TODO: Create a plugin runner helper to provide the correct
+		// package manager
+		body: source`
+			The simulator requires extra dependencies. Run the following command to install them.
+
+			~~~sh
+			npm install --save ${REQUIRED_DEPENDENCIES.join(" ")}
```

Comment:
Yes, there is a TODO just above that line:

```
// TODO: Create a plugin runner helper to provide the correct
// package manager
```

We have some related method in the manager (e.g. `manager.project.detectPackageManager()`), but they are not exposed to plugins. We would need to make a dedicated helper. Exposing those methods is tricky because we don't have access to the manager in the plugin runner.